packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -44,20 +44,6 @@ const StepperHeader = ({ has_target, has_real_account, has_currency, items, getC
                     </MobileWrapper>
                 </React.Fragment>
             )}
-            <DesktopWrapper>
-                {has_real_account && (!has_target || !has_currency) && (
-                    <div className='account-wizard__set-currency'>
-                        {!has_currency && (
-                            <p>
-                                <Localize i18n_default_text='You have an account that do not have currency assigned. Please choose a currency to trade with this account.' />
-                            </p>
-                        )}
-                        <Text as='h2' weight='bold' align='center'>
-                            <Localize i18n_default_text='Please choose your currency' />
-                        </Text>
-                    </div>
-                )}
-            </DesktopWrapper>
```

Comment:
I don't think we need to remove these content here, because in order to create a MF account, first, user needs to set currency for their MX account and then proceed to create MF account.
In this case, the modal header looks wrong!

Reply:
We will not use this modal for setting currency for MX/CR accounts thus there is no need for this content. We will use set_currency modal instead.